content/coc/index.md

```diff
@@ -0,0 +1,116 @@
+---
+layout: page
+title: "Code of Conduct"
+toc: true
+---
+
+The NYC Mesh is dedicated to fostering an environment in which everyone can participate in our meetups, installs, online spaces, and any other community event.
```

Comment:
The NYC Mesh?

Reply:
Huh, taking this as a cue to read up on some of our copy it looks like we prefer to style ourselves as "NYC Mesh" without the definite article. I'll go ahead and change that.

Why do we prefer "NYC Mesh" over "The NYC Mesh"?

Reply:
That’s how it’s always been :)

Also “the” sounds strange in my opinion.

Reply:
yeah, thinking about it we're just one of many NYC meshes, and happen we call ourselves "NYC Mesh".